### PR TITLE
Konstantin/appeals 48306

### PR DESCRIPTION
**Pull request by KonstantinShevtsov**
Resolves Add Toggle to AMA Eventing API (https://jira.devops.va.gov/browse/APPEALS-48306)

# Description
That PR adds the opportunity to turn off the ama events API using the new feature toggle: ama_eventing_disabled.

## Acceptance Criteria
- [x] Code compiles correctly

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/APPEALS-49509)

# Backend
## Database Changes
 None

## Integrations: Adding endpoints for external APIs
 None

### Test Coverage
Did you include any test coverage for your code? Check below:
- [X] RSpec

### Code Climate
Your code does not add any new code climate offenses? If so why?
- [X] No new code climate issues added

### CODE REVIEW

File: app/controllers/api/events/v1/decision_review_created_controller.rb [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # frozen_string_literal: true
 
 class Api::Events::V1::DecisionReviewCreatedController < Api::ApplicationController
+  include ApiV3FeatureToggleConcern
+
+  # Checks if API is disabled
+  before_action do
+    api_disabled?(:ama_eventing_disabled)
```
Comment (jtsangVA) [inline app/controllers/api/events/v1/decision_review_created_controller.rb:8]: rename to `disable_ama_eventing`

Comment (jtsangVA) [reply app/controllers/api/events/v1/decision_review_created_controller.rb:8]: change to !api_enable ?

File: app/controllers/api/events/v1/decision_review_created_controller.rb [hunk 2]
```diff
@@ -1,6 +1,23 @@
 # frozen_string_literal: true
 
 class Api::Events::V1::DecisionReviewCreatedController < Api::ApplicationController
+
```
Comment (jtsangVA) [inline app/controllers/api/events/v1/decision_review_created_controller.rb:4]: per linter, please remove extra empty line

**Review by jtsangVA — APPROVED** [review]

**Review by mattray-gov — COMMENTED** [review]
LGTM

**Pull request merged**